src/Microsoft.IdentityModel.Tokens/Encryption/EcdhKeyExchangeProvider.cs

```diff
@@ -116,7 +100,7 @@ public SecurityKey GenerateKdf(string apu = null, string apv = null)
             //The "apu" and "apv" values MUST be distinct when used (per rfc7518 section 4.6.2) https://datatracker.ietf.org/doc/html/rfc7518#section-4.6.2
             if (!string.IsNullOrEmpty(apu)
                 && !string.IsNullOrEmpty(apv)
-                && apu.Equals(apv, StringComparison.InvariantCulture))
+                && apu.Equals(apv, StringComparison.Ordinal))
```

Comment:
We found out that adding 'StringComparison.Ordinal', adds a performance penalty.
We should just leave it out.

Reply:
Removed StringComparison.Ordinal and added a supression message with the performance penalty as justification.